docker-compose.yml

```diff
@@ -106,6 +106,11 @@ services:
   memcached:
     <<: *restart_policy
     image: "memcached:1.6.9-alpine"
+    command: ["-I", "${SENTRY_MAX_EXTERNAL_SOURCEMAP_SIZE:-1M}"]
+    environment:
+      # Leaving the value empty to just pass whatever is set
+      # on the host system (or in the .env file)
+      SENTRY_MAX_EXTERNAL_SOURCEMAP_SIZE:
```

Comment:
Don't think you need this as the memcached image should be oblivious to any `SENTRY_` env variable :) Instead, you should move `SENTRY_MAX_EXTERNAL_SOURCEMAP_SIZE:` part to line 73, next to `x-sentry-defaults`' environment section

Reply:
fixed by https://github.com/getsentry/self-hosted/pull/2050/commits/4881f244bdc866f8e195b7371bc402d75ed46073